### ISSUE TITLE
Signing Service: Passing secrets in files or environment variables

### ISSUE THREAD

**Issue by cameel** [golemfactory/concent#671]
This is a part of the implementation of the "Concent Signing Service" blueprint (#597). This task depends on #633.

Currently Signing Service needs two secrets: Ethereum private key and Sentry DSN. The private key needs to be provided in a file and DSN directly on the command line. Let's make it more consistent. It should be possible to provide each secret in three ways:
- On the command line (`--<secret>` option)
- In an environment variable  (`--<secret>-from-env` option)
- In a file  (`--<secret>-path` option)

Only one option can be specified. Variable should be the default.

### Examples
``` bash
export ETHEREUM_PRIVATE_KEY=123-key-456
./signing-service.sh --ethereum-private-key-from-env
```

``` bash
echo "123-dsn-456" > /tmp/dsn.txt
./signing-service.sh --sentry-dsn-path=/tmp/dsn.txt
```

``` bash
./signing-service.sh --ethereum-private 123-key-456
```

**Comment by cameel** [comment]
Yes. It should contain the same thing you would put in the variable or on the command line.

**Comment by kbeker** [comment]
@cameel what about type of variables which will be stored as a env variables or variables in files? Should all of them be stored as `Base64` type? Or somehow different?

**Comment by cameel** [comment]
Depends on what you're storing. In case of keys obviously you can't use raw bytes on the command line or in the variable. So you need to use some form of encoding. base64 is a good choice because we're using it in other places too.

The stuff in file should be the same as used on the command line for consistency. So if you use base64, expect it even if the secret is passed in a file.

**Comment by bartoszbetka** [comment]
@cameel In case of environment variables, we have to wait for the `base64 --decode' variables at the script input, because the kubernetes secrets decode base64 for us.
